app/views/sessions/new.html.erb

```diff
@@ -33,15 +33,12 @@
 
     <%= f.text_field :username, :label => t(".email or username"), :autofocus => true, :tabindex => 1, :value => params[:username] %>
 
-    <div class="row">
-      <div class="col">
-        <%= f.label :password, t(".password"), :class => "form-label" %>
-      </div>
-      <div class="col text-end">
-        <small><%= link_to(t(".lost password link"), user_forgot_password_path) %></small>
-      </div>
+    <div class="col d-flex justify-content-between align-items-center">
+      <%= f.label :password, t(".password"), :class => "my-1" %>
```

Comment:
I think that class wants to be `my-2` to make the whole row the same height as before, and as the label for the username - as it is the label for password is closer to the field than the label for username is.

Reply:
Spot on, resolved.